docs/_documentations/troubleshooting.md

```diff
@@ -849,4 +857,149 @@ Command options are:
+
+- [Codewind output stream](#codewind-output-stream)
+- [Finding the extension logs](#finding-the-extension-logs)
+- [Build succeeded, but project is stuck in the Stopped state](#build-succeeded-but-project-is-stuck-in-the-stopped-state)
+- [No ESLint warnings or errors](#no-eslint-warnings-or-errors)
+- [Debug](#debug)
+
+## Codewind output stream
+
+The Codewind output stream is available in the VS Code editor. It logs `cwctl` commands together with their ouput.
+
+Check the Codewind output stream first when troubleshooting because it is particularly useful in helping you to debug unusual problems especially when starting Codewind. Some errors will also provide you with a button to open the Codewind output stream, for example:
+
+![Image of VS Code output with View button](images/cdt-vsc/output_view.png)<br>
+
+## Finding the extension logs
+
+If you report an issue, you will be asked to upload your logs.
+
+1. In VS Code, open **Help** > **Toggle Developer Tools**.
+2. Go to the **Console** tab.
+3. Enter *codewind.log* in the **Filter** box:
+<br>![Log location](images/cdt-vsc/logs-location.png)
+4. Upload the contents of the log file with your issue report.
+
+## Build succeeded but project is stuck in the Stopped state
+If your project fails to start, check the application logs for anything that might indicate the failure.
+If your application does not have problems, you can [check the Codewind logs](troubleshooting.html#check-the-logs).
+If the error persists, you can reset the project by disabling and re-enabling the project.
+
+## No ESLint warnings or errors
+You see no ESLint warning or errors for Node.js projects. Install the [ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and follow the instructions to activate the extension.
+
+## Debug
+### Liberty project remains in the Starting - Debug state
+Liberty servers started in **Debug** mode do not start until the debugger attaches. Run the `attach debugger` command] and the server starts. You can check the application logs to see if the server is starting.
+### Debugger attach fails with the message "Configured debug type "java" is not supported"
+Install and enable the [Java Extension Pack](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack).
+### Debugger fails to attach after restarting project into Debug mode
+Run the `attach debugger` command again. If the issue persists after a few attempts, restart the project in **Debug** mode a second time.
+
+***
+# Che troubleshooting
+
+When troubleshooting, check the project in Codewind to make sure the status is what you expect. For Codewind specific problem solving tips when using Che, see the following information.
+
+* [Check the Che logs](#check-the-che-logs)
+* [Solving common Che problems](#solving-common-che-problems)
+
+## Check the Che logs
+The logs are found in your Eclipse workspace under *.metadata/.log*.
```

Comment:
This is not accurate, those are the Eclipse IDE logs. You can just remove this section.

Reply:
Removed.